spec/statement.md

```diff
@@ -0,0 +1,177 @@
+# Statement
+
+## Table of Contents
+
+* [Definitions](#definitions)
+  * [Current indentation level](#current-indentation-level)
+  * [Document indentation character](#document-indentation-character)
+* [Syntax](#syntax)
+  * [ScssStatements](#scssstatements)
+  * [IndentedStatements](#indentedstatements)
+  * [Stylesheet](#stylesheet)
+  * [Block](#block)
+  * [Comments](#comments)
+    * [LoudComment](#loudcomment)
+    * [SilentComment](#silentcomment)
+    * [WhitespaceComment](#whitespacecomment)
+  * [Whitespace](#whitespace)
+  * [Indentation](#indentation)
+
+## Definitions
+
+### Current indentation level
+
+The *current indentation level* is the number of [documentation indentation
```

Comment:
shouldn't this be `document` instead of `documentation` ?

Reply:
Opened #3992 to fix. Thanks!